Earthfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:20.04
 WORKDIR /defs
 
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Do you need to set the timezone manually as well, like in these examples?

https://serverfault.com/questions/949991/how-to-install-tzdata-on-a-ubuntu-docker-image

Reply:
It defaults to UTC:

```
$ docker run --rm ubuntu:20.04 /bin/sh -c 'export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install tzdata && cat /etc/timezone'
Get:1 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]
Get:2 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]
Get:3 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [1381 kB]
Get:4 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Get:5 http://archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]
Get:6 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [786 kB]
Get:7 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [30.1 kB]
Get:8 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [828 kB]
Get:9 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages [1275 kB]
Get:10 http://archive.ubuntu.com/ubuntu focal/restricted amd64 Packages [33.4 kB]
Get:11 http://archive.ubuntu.com/ubuntu focal/multiverse amd64 Packages [177 kB]
Get:12 http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages [11.3 MB]
Get:13 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [1809 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [852 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [33.7 kB]
Get:16 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1109 kB]
Get:17 http://archive.ubuntu.com/ubuntu focal-backports/main amd64 Packages [50.8 kB]
Get:18 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [21.7 kB]
Fetched 20.3 MB in 5s (4149 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  tzdata
0 upgraded, 1 newly installed, 0 to remove and 3 not upgraded.
Need to get 295 kB of archives.
After this operation, 4033 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 tzdata all 2021e-0ubuntu0.20.04 [295 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 295 kB in 1s (274 kB/s)
Selecting previously unselected package tzdata.
(Reading database ... 4127 files and directories currently installed.)
Preparing to unpack .../tzdata_2021e-0ubuntu0.20.04_all.deb ...
Unpacking tzdata (2021e-0ubuntu0.20.04) ...
Setting up tzdata (2021e-0ubuntu0.20.04) ...

Current default time zone: 'Etc/UTC'
Local time is now:      Wed Jan  5 16:38:16 UTC 2022.
Universal Time is now:  Wed Jan  5 16:38:16 UTC 2022.
Run 'dpkg-reconfigure tzdata' if you wish to change it.

Etc/UTC    <-------- /etc/timezone gets set here
```